src/React.re

```diff
@@ -528,16 +555,16 @@ module Uncurried = {
     "useCallback";
 };
 
-type transitionConfig = {timeoutMs: int};
-
 [@bs.module "react"]
-external useTransition:
-  (~config: transitionConfig=?, unit) =>
-  (callback(callback(unit, unit), unit), bool) =
+external useTransition: unit => (bool, callback(callback(unit, unit), unit)) =
   "useTransition";
 
 [@bs.set]
 external setDisplayName: (component('props), string) => unit = "displayName";
 
 [@bs.get] [@bs.return nullable]
 external displayName: component('props) => option(string) = "displayName";
+
+[@bs.module "react"]
+external useDebugValue: (~format: 'value => string, 'value) => unit =
```

Comment:
Woops, I dislike the unit when the labelled arg is last. `useDebugValue: string` and `useDebugValueFormat: 'value, 'value => string` sounds good here?

Reply:
or just a wrapper with an idendity function by default
```
[@bs.module "react"]
external useDebugValue: ('value, ~format: 'value => string=?, unit) => unit =
  "useDebugValue";

let useDebugValue = (value, ~format=(a => a)) =>
  useDebugValue(value, ~format, ());
```

Reply:
what's the issue with the unit argument? It looks like `format` is optional so we can't do your first suggestion.

IMO the `unit` argument is idiomatic in OCaml for functions that have optional labeled arguments.

Reply:
There's no issue, just a preference and trying to avoid it.

Reply:
It's still more idiomatic in OCaml to have an optionally labelled argument despite of your personal preference, though.

I'd prefer a single function rather than 2. In Melange, the unit argument is ignored in the FFI.

Reply:
Yes, totally pushed the version with the unit